src/lib/crypto/snarky_tests/examples/range_gt.json

```diff
@@ -1 +1 @@
:"Generic","wires":[{"row":1,"col":3},{"row":4,"col":1},{"row":5,"col":0},{"row":3,"col":5},{"row":4,"col":4},{"row":3,"col":2},{"row":4,"col":6}],"coeffs":["00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000"]},{"typ":"Generic","wires":[{"row":4,"col":2},{"row":5,"col":5},{"row":6,"col":3},{"row":1,"col":0},{"row":5,"col":4},{"row":5,"col":1},{"row":5,"col":6}],"coeffs":["0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000"]},{"typ":"Generic","wires":[{"row":2,"col":1},{"row":6,"col":5},{"row":7,"col":0},{"row":5,"col":2},{"row":6,"col":4},{"row":6,"col":1},{"row":6,"col":6}],"coeffs":["0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000"]},{"typ":"Generic","wires":[{"row":6,"col":2},{"row":7,"col":1},{"row":7,"col":2},{"row":7,"col":3},{"row":7,"col":4},{"row":7,"col":5},{"row":7,"col":6}],"coeffs":["0100000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040"]}]}
```

Comment:
In these files, is it OK that coefficients appear as hex strings whereas the same file in #13592 are given as arrays of bytes?

Reply:
The updates to the examples JSON are displayed when you run the unit tests, explaining the difference in format. 

Using the same format as in #13592 results in errors.